src/test/java/seedu/guest/logic/commands/ResetRoomCleanCommandTest.java

```diff
@@ -0,0 +1,75 @@
+package seedu.guest.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.guest.logic.commands.CommandTestUtil.VALID_IS_ROOM_CLEAN_BOB;
+import static seedu.guest.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.guest.testutil.TypicalGuests.getTypicalGuestBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.guest.model.Model;
+import seedu.guest.model.ModelManager;
+import seedu.guest.model.UserPrefs;
+import seedu.guest.model.guest.Guest;
+import seedu.guest.testutil.GuestBuilder;
+
+
+
+
```

Comment:
Is there any reason for these extra newlines?

Reply:
No reason, will remove.